oar/cli/cmd_image_consistency_check.py

```diff
@@ -35,23 +35,27 @@ def image_consistency_check(ctx, build_number):
             )
         elif image_consistency_result == TASK_STATUS_INPROGRESS:
             logger.info(
-                "job[image-consistency-check] already triggered and in progress, not need to trigger again"
+                "job[image-consistency-check] already triggered and in progress, no need to trigger again"
             )
         else:
             nm = NotificationManager(cs)
             try:
-                build_url = jh.call_image_consistency_job()
+                block_status = jh.pre_check_build_queue("image-consistency-check")
+                if (block_status):
+                    logger.warning(f"a existed job is pending for image-consistency-check job, please trigger a new job later")
```

Comment:
@wewang58 the message is not meaningful, we need to let the user know, there is some pending jobs in the queue, your request cannot not be accepted directly, please try again later. 

`there is pending job in the queue, please try again later`